From d3a14f95c6aec6919dce9359f0e4f31aa5364976 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Sun, 31 Jul 2011 19:18:02 -0400
Subject: sh: Add module.h to arch/sh specific files as required.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
(cherry picked from commit db4e83957f961f9053282409c5062c6baef857a4)

Conflicts:

	drivers/sh/maple/maple.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 drivers/sh/intc/dynamic.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/sh/intc/dynamic.c b/drivers/sh/intc/dynamic.c
index a3677c9..5fea1ee 100644
--- a/drivers/sh/intc/dynamic.c
+++ b/drivers/sh/intc/dynamic.c
@@ -14,6 +14,7 @@
 #include <linux/irq.h>
 #include <linux/bitmap.h>
 #include <linux/spinlock.h>
+#include <linux/module.h>
 #include "internals.h" /* only for activate_irq() damage.. */
 
 /*
-- 
1.7.10

